website/views.py

```diff
@@ -164,7 +164,7 @@ def serialize_log(node_log, auth=None, anonymous=False):
         'action': node_log.action,
         'params': sanitize.unescape_entities(node_log.params),
         'date': utils.iso8601format(node_log.date),
-        'node': node_log.node.serialize(auth) if node_log.node else None,
```

Comment:
`original_node` is the node the log was about (formerly the first node on the log backrefs) and `node` is where the log sits currently. `original_node` can be equal to node when the action happened to node A and I am viewing node A's logs but if I am looking at a registration or fork, many times they will be different.

For example, I create a **Node**.  I now have one log, `project_created`.  I then register the node, **Registration**

`project_created` log on **Node**
- original_node =  **Node**
- node =  **Node**

`project_created` log on **Registration**
- original_node =  **Node**
- node = **Registration**

On the front end for the registration, the logs need to refer to the original node.  It's not going to be **Registration** was created, but **Node**. The original node.